client/app/domain/glue/add/domain-glue-add-or-edit.controller.js

```diff
@@ -44,7 +43,7 @@ angular.module("controllers").controller(
             if (!this.domain.glueRecordMultiIpSupported && model.ips.length > 1) {
                 valid = false;
             } else {
-                valid = !_.isEmpty(model.ips) && _.every(model.ips, (ip) => this.Validator.isValidIpv4(ip) || (this.domain.glueRecordIpv6Supported && this.Validator.isValidIpv6(ip)));
+                valid = !_.isEmpty(model.ips) && _.every(model.ips, (ip) => this.Validator.isValidIpv4(ip) || (_.get(this.domain, "glueRecordIpv6Supported", false) && this.Validator.isValidIpv6(ip)));
```

Comment:
Could be `this.domain.glueRecordIpv6Supported === true`

Reply:
@FredericEspiau if this.domain is undefined, it will be broken ^.^

If we use these solution, plz check this.domain before :)